nginx/nginx.conf

```diff
@@ -89,8 +90,8 @@ http {
 			proxy_pass http://relay;
 		}
 		location ^~ /js-sdk/ {
-			autoindex on;
-			root  /var/www/js-sdk;
+			root /var/www/;
+			add_header Access-Control-Allow-Origin *; # It is recommended to change this to your sentry (in most cases same as system.url-prefix)
```

Comment:
nit: Would put the comment above the line for readability.

Reply:
This has not been fixed yet.